website/docs/docs/deploy/deploy-environments.md

```diff
@@ -186,6 +190,46 @@ This section allows you to determine the credentials that should be used when co
 </WHCode>
 
 
+## Staging environment <Lifecycle status='beta' />
+
+:::note
+Currently in limited availability beta. Contact support or your account team if you're interested in beta access.
+:::
+
+Staging environments are useful ways to grant developers access to deployment workflows and tools while controlling access to production data. They are configured with their own long-living branch (for example, `staging`) that may be very similar to `main` in many ways while potentially limiting the data the developers can access. 
+
+Ideally, the workflows would move upstream from the Development environment -> Staging environment -> Production environment with developer branches feeding into the staging branch, then ultimately `main`. In many cases, the `main` and `staging` branches will be identical after a merge and remain until the next batch of changes from the `development` branches are ready to be elevated. We recommend setting branch protection rules on `staging` similar to `main`.
+
+### Create a staging environment 
+
+In the dbt Cloud, navigate to **Deploy** -> **Environments** and then click **Create Environment**. Select **Deployment** as the environment type. The option will be greyed out if you already have a development environment.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/create-staging-environment.png" width="85%" title="Create a staging environment" />
+
+
+Follow the steps outlined in [deployment credentials](#deployment-connection) to complete the remainder of the environment setup.
+
+We recommend that the data warehouse credentials be for a dedicated user or service principal.
+
+### Why use a staging environment
```

Comment:
I'd expect to see the "why use this feature" before "Create a staging environment." Also, I'm not convinced this needs to be its own section. These [4 lines](https://github.com/dbt-labs/docs.getdbt.com/pull/5267/files#diff-b3b10e4b30d41964b19ae127ca5e8a640eb6000e03de275b6bec4c21b03ab142R216-R218) might fit better after the paragraph beginning with "[Ideally, the](https://github.com/dbt-labs/docs.getdbt.com/pull/5267/files#diff-b3b10e4b30d41964b19ae127ca5e8a640eb6000e03de275b6bec4c21b03ab142R201)"